style.css

```diff
@@ -22,71 +21,508 @@ ul {
   list-style-type: none;
 }
 
+header {
+  position: fixed;
+  width: 1440px;
+  height: 72px;
+  left: 0.001px;
+  top: 0.001px;
+  background: #ffff;
+}
+
+.nav-bar {
+  height: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 130px;
+  padding-top: 20px;
+  padding-right: 130px;
+}
+
+nav button {
+  display: none;
+}
+
+.nav-links {
+  display: flex;
+  align-items: center;
+  gap: 4rem;
+}
+
+.nav-links a {
+  color: #344563;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.intro {
+  width: 1440px;
+  height: fit-content;
+  left: 0.001px;
+  top: 1px;
+  background: #ffff;
+  border-bottom-left-radius: 70px;
+}
+
 .container {
-  display: grid;
-  margin: 0;
-  background-image: url("images/header-shapesmobile@2x.svg");
+  background-image: url("/images/h.png");
   background-repeat: no-repeat;
   background-size: cover;
+  margin-top: 72px;
+  height: 871px;
+  margin-bottom: 76px;
+  margin-left: 24px;
+  margin-right: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
 }
 
 .introduction {
-  font-family: "Poppins", sans-serif;
+  width: 920px;
+  height: 122px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 60px;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.37px;
+  color: #091e42;
+}
+
+.content {
+  width: 920px;
+  height: 98px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #344563;
+}
+
+.connect {
+  width: fit-content;
+  height: 24px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  align-items: flex-start;
+}
+
+.social-image ul {
+  padding-top: 20px;
+  display: flex;
+  gap: 20px;
+  list-style: none;
+  flex-direction: row;
+  margin-left: 34px;
+}
+
+.posts {
+  background: #ffff;
+  border: 1px solid #dfe1e6;
+  border-radius: 24px;
+  height: 496px;
+  width: 1156px;
+  margin: 142px;
+}
+
+.post {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 22px;
+}
+
+.image {
+  width: 544px;
+  height: 448px;
+  left: 24px;
+  top: 24px;
+  padding-top: 20px;
+  margin-top: 10px;
+  background-image: url("/images/Snapshoot\ Portfolio.png");
+}
```

Comment:
- [ ] You make significant progress but as the previous reviewer mentioned you should follow the design like the Figma template, the image for the card does not appear. To show it change the name of the image for any name but don't leave space in it like `snapshot-portfolio.png`